### DISCUSSION TITLE
[Solved] 5.2 Single project update DispatcherQueue error?

### DISCUSSION THREAD

**Discussion by hawkerm** [unoplatform/uno#16643] [Q&A] [answered]
Trying to migrate an Uno project to the latest single-project 5.2 update with the Uno.Sdk.

It wasn't as clear to me where I should put general package references (within the app's csproj itself?), like the CommunityToolkit 8.1-rc.

However, the current errors I'm getting when building are around the DispatcherQueue reference:

```
error CS0012: The type 'DispatcherQueue' is defined in an assembly that is not referenced. You must add a reference to assembly 'Uno.UI.Dispatching, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null'
```

Seems like it's picking up the Uno assemblies over the Windows ones? (Just trying to initial build/run in the standard Windows App SDK Packaged profile.)

**Comment by hawkerm** [comment, marked as answer]
Oh, I answered my own question, as I forgot the Windows Community Toolkit 8.1 requires `-windows10.0.22621` now...

Then I just had to fiddle in VS a bit as it switched the selected target framework in the run drop-down for the configuration. So, it works now!

**Reply by jeromelaban** [reply]
Interestingly, with recent changes in the WCT, I'd have thought that the error would have not involved Uno. Do you know which version of uno is used to build?

**Reply by agneszitte** [reply]
@hawkerm note that our documentation regarding WCT will be updated soon.
Related issue with the details: https://github.com/unoplatform/uno/issues/16872

@jeromelaban for this discussion, is it related to [Xaml Internal Error WMC9999 on when updating to v8.1-rc](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/issues/185#top)#185 ?

**Reply by jeromelaban** [reply]
Yes it is.
